### PR TITLE
Update code syle documentation

### PR DESCRIPTION
**Pull request by bryteise**
Add details for if statement bracket use and function and variable
names.

### CODE REVIEW

File: HACKING [hunk 1]
```diff
@@ -45,6 +48,9 @@ Unacceptable:
 Minimise your use of "goto"'s, and test every code path is reached. Also
 ensure *every* if/else, etc, even if single line, is wrapped in curly braces.
 
+if statements should always use brackets regardless of the number of lines in
```
Comment (busykai) [inline HACKING]: if => conditional (for `if` and `else`)?

File: HACKING [hunk 2]
```diff
@@ -45,6 +48,9 @@ Unacceptable:
...
+the consequent.
```
Comment (busykai) [inline HACKING]: consequent "block"?

Comment (bryteise) [reply HACKING]: Ah yes, thanks!

File: HACKING (hunk 1 above)
Comment (bryteise) [reply HACKING]: Yes, should I say if and if then here instead?

**Comment by bryteise** [comment]
I'll add the clang-format bits as well.

File: HACKING (hunk 1 above)
Comment (busykai) [reply HACKING]: Yes. You meant `if` and `else`, I assume.

Comment (bryteise) [reply HACKING]: Indeed I did =).

**Comment by bryteise** [comment]
Updated with changes from comments.

**Pull request merged**